src/snarky-class-spec.json

```diff
@@ -0,0 +1,316 @@
+[
```

Comment:
Is there an easy way we can avoid checking this in since it’s auto-generated? Make generating this the first step of the build or something. If it’s too much effort, we can do it later though.

Reply:
Yeah that should work!

Reply:
At least until I get to make the bode build fast enough that this becomes a noticeable bottleneck ^^

Reply:
Updated the PR!